### ISSUE TITLE
Keyboard stops working a lot

### ISSUE THREAD

**Issue by CalbernandHowbe** [Sentero-esp12/Multi-Premoves-Mouse-Keyboard#10]
Normally, sometimes this happened with the old extension, that you needed to refresh every once in a while in between games to get the kb to work otherwise the pieces don't move, however it is happening a LOT more frequently now, like once every game, and in the middle of the game, and it is necessary to refresh to get it working again, making the extension currently unusable

**Comment by Sentero-esp12** [comment]
What settings do you have for:

- How do you move pieces?
- Promote to Queen automatically
- Castling method
- Piece animation

**Comment by CalbernandHowbe** [comment]
Both,
Always autoqueen,
Move king onto rook,
Normal

**Comment by CalbernandHowbe** [comment]
I reverted my settings to what i had before (listed above) and played some casual games and it worked completely fine. However when playing vs stockfish level 1 it requires a reload sometimes.

**Comment by CalbernandHowbe** [comment]
And I tried it in a tournament just now and it didnt work at all.

**Comment by Sentero-esp12** [comment]
Thanks for testing, 
It seems that most players experience the same (and it started happening recently). 
Are you playing with the warning on the top disabled or enabled?

**Comment by CalbernandHowbe** [comment]
If you are talking about the warning which says "Lichess Multi Premoves is debugging this browser' then it's enabled

**Comment by CalbernandHowbe** [comment]
And also, what seems to happen is there is a "period" after playing a game in the lobby when it works fully, and it works until you play a game in the tournament, and then it never works for a period until you play another game in the lobby. weird

**Comment by Sentero-esp12** [comment]
> and then it never works for a period until you play another game in the lobby
Do you mean after playing a game in a tournament it stops working completely (nothing moves, no circle)? Or works partially, until you play another game in a lobby?

**Comment by CalbernandHowbe** [comment]
Yes, because when I play a game in a tournament it NEVER seems to work. And then I have to restart my browser to get it to work again.

**Comment by CalbernandHowbe** [comment]
So:
Lobby games: works perfectly
Games vs stockfish: works somewhat but requires a refresh frequently inbetween games
Tournament games: never

**Comment by Sentero-esp12** [comment]
When you play do selected squares appear slightly shifted? Also could you try closing Chrome and running with Terminal: /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --silent-debugger-extension-api

I'm wondering if the warning on the top could be the reason

**Comment by Bobkingmen3** [comment]
After refreshing the tab prior to each tournament game this problem seems to be eliminated. Version 0.0.0.1
